app/markets/HistorySection.tsx

```diff
@@ -1,7 +1,7 @@
 import { ActivityTable } from '@/app/markets/ActivityTable';
 import { Icon, TabBody, TabGroup, TabHeader, TabPanel, TabStyled } from '@swapr/ui';
 
-export const HistorySection = () => {
+export const HistorySection = ({ id }: { id: string }) => {
   return (
     <div className="p-2">
```

Comment:
I know it's outside your changes, but maybe we can drop this `div` element and move the `className` into `<TabGroup>`

Reply:
`TabGroup` is `Fragment`